content/00.front-matter.md

```diff
@@ -0,0 +1,12 @@
+# Manubot Rootstock: Manuscript Title
+
+## Authors
+
+{% for author in authors %}
++ **{{author.full_name}}**<br>
+  [\@{{author.github_username}}](https://github.com/{{author.github_username}}) ·
+  [{{author.orcid}}](https://orcid.org/{{author.orcid}}) ·
+  {{author.affiliations}} ·
+  Funded by {{author.funders}}
```

Comment:
In a full manuscript, this would be better suited for a dedicated funding section in the "back matter".  For this example manuscript, I think it is fine to leave here.  Users should be able to figure out how to loop over the authors again if they want to move the funding information.